Automattic/jetpack
# AI Assistant: Add long sentences feature to proofread

Fixes #38313

## Proposed changes:

* Adds the long sentence feature

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add a sentence with more than 20 words (ending with ., ? or !)
* Check that the sentence is highlighted
